examples/chef/chef.py

```diff
@@ -894,6 +897,13 @@ def main() -> int:
             else:
                 linux_args.append("chip_inet_config_enable_ipv4=false")
 
+            if options.enable_lit_icd:
+                linux_args.append("chip_enable_icd_server = true")
+                linux_args.append("chip_subscription_timeout_resumption = true")
+                linux_args.append("chip_icd_report_on_active_mode = true")
+                linux_args.append("chip_enable_icd_lit = true")
+                linux_args.append("chip_enable_icd_dsls = true")
```

Comment:
Does the contact sensor implement this feature?

Reply:
The question is does the contact-sensor require this feature in the sense do we need to enable it? I don't think the current contact sensor implementation uses but i might be wrong. 

The feature can be enabled without it being actually used. You can ignore the comment.

Reply:
Got it. Thank you!!